config/environments/production.rb

```diff
@@ -142,4 +142,20 @@
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
 
   config.cip_resource_bucket = "https://paas-s3-broker-prod-lon-ac28a7a5-2bc2-4d3b-8d16-a88eaef65526.s3.amazonaws.com"
+
+  # From https://github.com/paper-trail-gem/paper_trail/wiki/Setting-whodunnit-in-the-rails-console
+  console do
+    PaperTrail.request.whodunnit = lambda {
+      @paper_trail_whodunnit ||= begin
+                                   email = nil
+                                   until email.present?
+                                     # rubocop:disable Rails/Output
+                                     puts "Enter your email address for PaperTrail"
+                                     # rubocop:enable Rails/Output
+                                     email = gets.chomp
+                                   end
+                                   email
+                                 end
+    }
+  end
```

Comment:
Can we not do that in development?

Or perhaps in any non-prod environment?

Reply:
Not a _huge_ fan of this. I think it might get a bit annoying. We can trace back SSH sessions to users with PaaS events.

Reply:
@kicferk1 this is production only a the moment.

@jamesrose there have been a few times where I have been trying to figure out how a record got into the state it was, and only managed by going through Jira to find the support ticket. It's worth noting this will only pop up when you start trying to modify records

Reply:
discussed